testing/citest/tests/kube_v2_smoke_test.py

```diff
@@ -311,27 +376,36 @@ def scenario(self):
   def test_a_create_app(self):
     self.run_test_case(self.scenario.create_app())
 
-  def test_b_deploy_manifest(self):
+  def test_b1_deploy_manifest(self):
     self.run_test_case(self.scenario.deploy_manifest('library/nginx'),
                        max_retries=1,
                        timeout_ok=True)
 
-  def test_c_update_manifest(self):
+  def test_b2_update_manifest(self):
     self.run_test_case(self.scenario.deploy_manifest('library/redis'),
                        max_retries=1,
                        timeout_ok=True)
 
-  def test_d_undo_rollout_manifest(self):
+  def test_b3_undo_rollout_manifest(self):
     self.run_test_case(self.scenario.undo_rollout_manifest('library/nginx'),
                        max_retries=1)
 
-  def test_e_scale_manifest(self):
+  def test_b4_scale_manifest(self):
     self.run_test_case(self.scenario.scale_manifest(), max_retries=1)
 
-  def test_f_patch_manifest(self):
+  def test_b5_patch_manifest(self):
```

Comment:
We really have to figure out a proper test ordering mechanism.

Reply:
Heh